frontend/tests/setup-test-environments.js

```diff
@@ -28,23 +25,22 @@ export const deploymentConfigs = {
 
 // CLI --------------------------------------------------------------------------------------------------------------
 // https://playwright.dev/docs/test-parameterize
-// let envsString = 'local-dev-prod';
-// let envsString = 'local';
-let envsString = DEPLOYMENT;
-
-// replacing process.env with import.meta.env for vite migration; see https://www.freecodecamp.org/news/how-to-migrate-from-create-react-app-to-vite/
+// Usable like: ENVIRONMENTS=dev-prod yarn test:e2e
 
+// - Determine which environments to run tests on
+let envsString = DEPLOYMENT;
 if (process.env.ENVIRONMENTS) {
   envsString = process.env.ENVIRONMENTS;
 }
-
 const envs = envsString.split("-");
 export let selectedConfigs = {};
 for (let key in deploymentConfigs) {
   if (envs.includes(key)) {
     selectedConfigs[key] = deploymentConfigs[key];
   }
 }
+
+// todo?: replace process.env with import.meta.env for vite migration; see https://www.freecodecamp.org/news/how-to-migrate-from-create-react-app-to-vite/
```

Comment:
## Vite changes?
I saw this comment, along with a bunch of commented out code.

@Sigfried We're using Vite, right? But yet these playwright tests still run just fine without these changes you commented out. Are things fine then? Can I remove this commented out code? If not, why is it needed?